tests/test_bundle.py

```diff
@@ -984,6 +984,12 @@ def test_patch_name_collision(self):
         )
         self.assertEqual(res.status_code, requests.codes.bad_request)
 
+    def test_enumeration_bundles(self):
+        bundle_uuid, bundle_version = self._put_bundle()
```

Comment:
This test may take a long time to run if a larger number of bundles are present in the test bucket. Consider using a random UUID of the format `00000000-xxx-xxxx-xxxx-xxxxxxxxxxxx`, where the x's are random hex digits. This should help the bundle appear near the beginning of the results.

Listing should halt when:
1. The bundle is found
2. a non-zeros prefix is encountered in the UUID (this will occur upon test failure.

Reply:
This change should probably be made in the smoketest as well.